app/frontend/wallet/blockchain-explorer.ts

```diff
@@ -119,30 +119,31 @@ const blockchainExplorer = (ADALITE_CONFIG) => {
   }
 
   function prepareTxHistoryEntry(tx: CaTxEntry, addresses: string[]): TxSummaryEntry {
-    const outputTokens: Token[][] = []
-    const inputTokens: Token[][] = []
+    const outputTokenBundle: TokenBundle[] = []
+    const inputTokenBundle: TokenBundle[] = []
 
     let effect = 0 //effect on wallet balance accumulated
     for (const [address, amount] of tx.ctbInputs || []) {
       if (addresses.includes(address)) {
         effect -= +amount.getCoin
-        const formatedInputTokens = amount.getTokens.map((token) => formatToken(token, -1))
-        inputTokens.push(formatedInputTokens)
+        const formatedInputTokenBundle = amount.getTokens.map((token) => parseToken(token))
```

Comment:
I guess this should now be `parsedInputTokenBundle` after formatToken has been renamed